snitun/server/run.py

```diff
@@ -151,6 +151,14 @@ def __init__(
         self._server: Optional[socket.socket] = None
         self._poller: Optional[select.epoll] = None
 
+    @property
+    def peer_counter(self) -> int:
+        """Return number of active peer connections."""
+        total = 0
+        for worker in self._workers:
+            total += worker.peer_size
+        return total
```

Comment:
```suggestion
        return sum(worker["peer_size" for worker in self._workers)
```
This can be simplified